tools/retest/main.go

```diff
@@ -23,75 +26,180 @@ func main() {
 	handleError(err)
 	log.Printf("Found %d PRs", search.GetTotal())
 
+loop:
 	for _, pr := range search.Issues {
 		prNumber := pr.GetNumber()
-		log.Printf("Processing #%d", prNumber)
-		comments, _, err := client.Issues.ListComments(ctx, "stackrox", "stackrox", prNumber, nil)
+		log.Printf("#%d processing...", prNumber)
+		prDetails, _, err := client.PullRequests.Get(ctx, S, S, prNumber)
 		handleError(err)
```

Comment:
Here and elsewhere inside the loop I think we should continue with the next issue, rather than panic, because a problematic one could prevent doing any further work.